smack-extensions/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatManager.java

```diff
@@ -282,6 +282,9 @@ public boolean isServiceEnabled(Jid user) throws NoResponseException, XMPPErrorE
      * Returns a Set of the rooms where the user has joined. The Iterator will contain Strings where each String
      * represents a room (e.g. room@muc.jabber.org).
      *
+     * Note: A more modern way to determine which MUCs the user might be joined in on other clients is using
+     * {@link BookmarkManager#getBookmarkedConferences()}.
```

Comment:
Could you fully qualify `BookmarkManager` here so that the import statement becomes unnecessary?

Reply:
I wasn't sure, whether or not the import statement would do harm, but sure, I can remove it :)